components/Header.tsx

```diff
@@ -11,8 +11,13 @@ export default function Header() {
 
 const StyledHeader = styled.header`
   position: sticky;
-  top: 2vh;
+  top: 0;
+  z-index: 1;
+  margin-left: 2rem;
+  margin-right: 2rem;
   text-align: center;
+  background-color: black;
+  border: 1px solid white;
 `;
```

Comment:
remove the `background-color: black` as it's cover your heading.
or set color to white for example.
I think the border property is only for dev reasons?

Reply:
yes the border property is only for dev reasons and the color is white as default. I set the bk color on black, because the bk color of the whole app is black. If I don´t do it, it is transparent and I can see how the content scroll through the header 🤷🏽